### ISSUE TITLE
Better Debug documentation for TypeScript

### ISSUE THREAD

**Issue by danielniccoli** [microsoft/vscode#158596]
I'm reading through https://code.visualstudio.com/docs/languages/typescript with the eyes of a TypeScript beginner and I believe [Debug Server-Side](https://code.visualstudio.com/docs/languages/typescript#_debug-server-side) should be improved.

It says:

> Debug Node.js in VS Code using the built-in debugger. Setup is easy and there is a [Node.js debugging tutorial](https://code.visualstudio.com/docs/nodejs/nodejs-tutorial#_debug-your-express-app) to help you.

There are two things I would criticise:

1. If setup is easy, I'd expect that one of two things would work:
    - Start debugging by pressing <kbd>F5</kbd>
    - Creating a default launch.json file with the before mentioned Node.js configuration.
    Both achieve the same thing, with the difference then one stores the configuration in a file. But both call `node.exe ./helloworld.ts`, which naturally results in a SyntaxError. I would not call this "easy" in a documentation that is clearly targeted towards absolute beginners on "Hello World" level.

2. The link to the "[Node.js debugging tutorial](https://code.visualstudio.com/docs/nodejs/nodejs-tutorial#_debug-your-express-app)" leads to a tutorial about how to debug a Express app. There are two things problematic with that:
    - The tutorial does not mention TypeScript compilation or debugging at all.
    - It doesn't help in the immediate attempt to setup a *TypeScript* debugger. Instead, it potentially overloads the reader with information about a framework that they are potentially unable to contextualise.

To improve the documentation, I would suggest that a basic or minimal working example is provided, so that after reading this tutorial, he is not only able to compile a Hello World, but also to set a break point, start the debugger and see the "Hello World" being printed to the server-side console.

**Comment by weinand** [comment]
I agree that "TypeScript debugging" is not easily to find in [TypeScript in Visual Studio Code](https://code.visualstudio.com/docs/languages/typescript).

The "Debugging" section links to generic debugging pages.

But scrolling down to the "Next Steps" section has a [Debugging TypeScript](https://code.visualstudio.com/docs/typescript/typescript-debugging) link.

I don't understand why this link is not more prominently located in the "Debugging" section.
